Sources/Internal/VolumeBarWindow.swift

```diff
@@ -42,6 +42,7 @@ internal final class VolumeBarWindow: UIWindow {
 		systemVolumeView.isHidden = false
 		systemVolumeView.clipsToBounds = true
 		systemVolumeView.showsRouteButton = false
+		systemVolumeView.alpha = 0.0001
```

Comment:
I saw the AirPlay audio picker in the Xcode view debugger so added this just in case to make sure the system volume slider definitely doesn't ever show up. 👀